### DISCUSSION TITLE
awesome-pocketbase updates

### DISCUSSION THREAD

**Discussion by benallfree** [pocketbase/pocketbase#1407] [Show and tell]
[awesome-pocketbase](https://github.com/benallfree/awesome-pocketbase) is a curated list of community-maintained PocketBase projects and resources.

Thanks to user contributions, we now have nearly 40 resources.

Please check it out and if you have anything you'd like to add, send a PR or open an issue.

**Comment by deavmi** [comment]
I just submitted a PR for [libpb](https://github.com/benallfree/awesome-pocketbase/pull/5)! :open_hands:
